src/example.md

```diff
@@ -1,6 +1,9 @@
-% Proof of Existing Pandoc Feature
-% Kurt Pfeifle
-% May 25th, 2015
+---
+title: "Proof of Existing Pandoc Feature"
+author: "Kurt Pfeifle"
```

Comment:
Who is this? could you put "John Doe" to emphasize that it is an example, or does this person really exist and you are giving credit?

Reply:
This person really exists, I'm giving them credit, and the link to their post is https://tex.stackexchange.com/questions/101717/converting-markdown-to-latex-in-latex/246871#246871 (this link is featured in the README.md)